plugins/source/aws/policies_v1/policy.sql

```diff
@@ -12,3 +12,4 @@ END AS "execution_time"  \gset
 \ir foundational_security/policy.sql
 \ir public_egress/policy.sql
 \ir publicly_available/policy.sql
+\ir imds_v2/policy.sql
```

Comment:
Not sure if we want to run this policy on every invocation...

Reply:
I'm open to either running it at invocation of the broader policies_v1 policy or standalone within imds_v2.  Went back and forth on that decision and ultimately, I initially left it in under policies_v1/policy.sql for completeness of running the rest of the queries, including public_egress, publicly_available, etc.